### DISCUSSION TITLE
OrchardRazorHelperExtensions Help

### DISCUSSION THREAD

**Discussion by TaylorEdm** [OrchardCMS/OrchardCore#7130] [Q&A]
I am attempting to use the Orchard Razor Helper Extension in a Orchard Core Razor theme. In the [documentation](https://docs.orchardcore.net/en/dev/docs/reference/modules/Media/) about the Orchard Media extension, Format is an optional parameter, but I don't see it in the Razor Helper. Is this only available in the liquid?

**Comment by agriffard** [comment]
Which version of Orchard Core are you using?
A pull request has been merged yesterday with this new parameter but only if you use the latest dev version.

**Reply by TaylorEdm** [reply]
I am using version 1.0.0-rc2-12450 .  I think it was released in June.
